tests/js/utils.js

```diff
@@ -99,10 +100,11 @@ export const createTestRegistry = () => {
  *
  * @param {?Object}   props          Component props.
  * @param {?Function} props.callback Function which receives the registry instance.
+ * @param {?Array}    props.features Feature flags to enable for this test registry provider.
```

Comment:
Since we're updating this would you please update the doc formatting for the params to use the correct optional syntax rather than nullable?

Reply:
Ah, yes, I always mix them up 😓